docs/emodb-example.rst

```diff
@@ -254,24 +258,47 @@ and assign the information to it.
         description='Six basic emotions and neutral.',
     )
     db.schemes['confidence'] = audformat.Scheme(
-        audformat.define.DataType.FLOAT,
+        'float',
```

Comment:
According to the type hints we do not support string. I created https://github.com/audeering/audformat/issues/269 to discuss if we should include it.